apps/andelabs.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+	dataTypes: function(argument){
+		switch (typeof (argument)){
+			case 'number':
+				return getNumberResult(argument); break;
+			case 'string':
+				return argument.length; break;
+			case 'object':
+				if (argument === null){
+					return 'no value'; break;
+				}
+				else{
+					return argument[2]; break;
+				}
+			case 'undefined':
+				return 'no value'; break;
+			case 'boolean':
+				return argument; break;
+			case 'function':
+				return argument(true); break;
+		}
+	},
+
+	getPrimes: function(number){
+		if (typeof(number) == 'number' && number >= 2){
+	        var sieve = [];			
+	        var i=2;
+
+	        top: while(i<=number){
+	            var isPrime = true;
+	            var j = 0;
+
+	            i++; 
+	            while (j<sieve.length && Math.sqrt(i-1)>=sieve[j]){
+	                if ((i-1) % sieve[j] === 0) 
+	                    continue top;
+	                j++;
```

Comment:
I fixed the consistency issues. Would create a new branch to refactor code and possibly deliver a less complicated code.